src/CMakeLists.txt

```diff
@@ -18,10 +18,10 @@ set(BITBUCKET_BASE_URL
     "${MADLIB_REDIRECT_PREFIX}https://bitbucket.org"
     CACHE STRING
     "Base URL for Bitbucket projects. May be overridden for testing purposes.")
-set(GITHUB_MADLIB_BASE_URL
+set(EIGEN_BASE_URL
```

Comment:
Since this is Eigen specific link, please include the `eigen/archive` part from line 55 in the URL itself.  That way it's used specifically for that purpose.